pgs_web/constants.py

```diff
@@ -129,3 +130,57 @@
     'OTH': 'Additional Diverse Ancestries',
     'NR' : 'Ancestry Not Reported'
 }
+
+PGS_CONTRIBUTORS = [
+    {'name': 'Sam Lambert','group': ['curators']},
+    {'name': 'Laurent Gil','group': ['hdruk']},
+    {'name': 'Benjamin Wingfield','group': ['ebi']},
+    {'name': 'Florent Yvon','group': ['inouye']},
+    {'name': 'Aoife McMahon', 'group': ['curators','nhgri-ebi']},
+    {'name': 'Santhi Ramachandran','group': ['nhgri-ebi']},
+    {'name': 'Elizabeth Lewis','group': ['nhgri-ebi']},
+    {'name': 'Laura Harris','group': ['nhgri-ebi']},
+    {'name': 'Helen Parkinson', 'group': ['spot']},
+    {'name': 'Richard Houghton','group': ['hdruk']},
+    {'name': 'Prof. John Danesh','group': ['hdruk']},
+    {'name': 'Michael Inouye', 'group': ['inouye']}
+]
+    
+PGS_GROUPS = {
+    'inouye': {
+        'name': 'Inouye Lab',
+        'url': 'https://www.inouyelab.org/'
+    },
+    'curators': {
+        'name': 'PGS Catalog Data Curators'
+    },
+    'hdruk': {
+        'name': 'Health Data Research UK, Cambridge',
+        'url': 'https://www.hdruk.ac.uk/about-us/our-locations/health-data-research-uk-hdr-uk-cambridge/'
```

Comment:
We could use the EBI and HDR-UK urls from `USEFUL_URLS` to avoid duplication

Reply:
Yes, I updated